R/report_descriptives.R

```diff
@@ -0,0 +1,18 @@
+# A funtion to take a seminr model and return item and construct descriptives
+get_descriptives <- function(seminr_model, na.rm = TRUE) {
+  #items
+  item_descriptives <- get_desc(seminr_model$data, na.rm = na.rm)
+  item_correlations <- cor(seminr_model$data)
+  #constructs
+  construct_descriptives <- get_desc(seminr_model$construct_scores, na.rm = na.rm)
+  construct_correlations <- cor(seminr_model$construct_scores)
+  class(item_descriptives) <- append(class(item_descriptives),"table_output")
```

Comment:
could you make it a practice to put a space after all commas?

Reply:
I will do. I have ran through the code and I believe removed this problem.